llarp/exit/session.cpp

```diff
@@ -53,10 +53,12 @@ namespace llarp
     bool
     BaseSession::ShouldBuildMore(llarp_time_t now) const
     {
+      if(BuildCooldownHit(now))
+        return false;
       const size_t expect = (1 + (numPaths / 2));
       // check 30 seconds into the future and see if we need more paths
       const llarp_time_t future = now + 30s + buildIntervalLimit;
-      return NumPathsExistingAt(future) < expect && !BuildCooldownHit(now);
+      return NumPathsExistingAt(future) < expect;
     }
```

Comment:
This is functionally equivalent, other than `BuildCooldownHit()` being the last checked condition instead of the first.

Reply:
yea, the real check is the next hunk as i understand it.

Reply:
(this was some cleanup)